Java/History/Evolution of Java/README.md

```diff
@@ -1 +1,39 @@
+# Evolution of Java
+
+Java was conceived by James Gosling, Patrick Naughton, Chris Warth, Ed Frank, and Mike Sheridan at Sun Microsystems, Inc in 1991. Basic idea behind the design of the language was that it is
+1. Simple
+2. Secure
+3. Portable
+4. Object oriented
+5. Robust
+6. Multi-threaded
+7. Architecture-neutral
+8. Interpreted
+9. High Performance 
+10. Distributed
+11. Dynamic
+
+While the first version of java was nothing short of revolutionary, it still managed to grow at a rapid rate. Each upgrade in the java consisted of major upgrades which reconfigured the way things were handled. The following timeline of upgrades breifly defines the evolution of java through the years.
+
+| Java Version | Year of release | Changes |
+|:------------:|----------------:|---------|
+| Java 1.0     |            1996 |    -    |
+| Java 1.1     |            1997 | It redefined the the way events were handled and reconfigured many features of initial release. It also rendered several features obselete|
+| J2SE 1.2     | 1998| It added several new features such as Swing and Collections Framework and it also enhanced JVM. It deprecated several functions from the thread class including ```suspend()```, ```resume()``` and ```stop()```.|
+| J2SE 1.3     | 2000 | For the most part, it added to the existing functionality and "tightened up" the development environment.|
+| J2SE 1.4     | 2002 | This release contained several important upgrades, enhancements, and additions. It added the new keyword assert, chained expections and a channel based I/O subsystem. It enhanced the collections framework and the networking classes.|
+| J2SE 5       | 2004 | J2SE 5 fundamentally expanded the scope, power and range of the language. The major new features included- generics, annotations, autoboxing and auto-unboxing, enumerations, enhanced for each style for loop, variable length arguments, static imports, formatted I/O, concurrency utilities. These functionalities added an entirely new dimension to java programming.|
+| Java SE 6 | 2006 | While Java SE 6 did not add any new features, it provided enhancement to API libraries, added several new features and offered improvements to the runtime.|
+| Java SE 7 | 2011 | It addressed several small features such as, a string could now be used in a switch statement, introducing binary integer literals, underscores in numeric literals, an expanded try statement, type reference when constructing a generic instance, enhanced exception handling  and better type checking.|
+| Java SE 8 | 2014 | It added an all new GUI library JavaFx along with a far-reaching feature, lambda expressions. It also included Date and Time API.|
```

Comment:
Add LTS label to all applicable version. Also, did you forget about the great stream api introduced in JDK 8. I mean its hurting 💀